<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>totalwidge</class>
 <widget class="QWidget" name="totalwidge">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>422</width>
    <height>416</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="l2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>90</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>计算方式：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="box1">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>50</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>商业贷款/公积金贷款</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>组合型贷款</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pBot1">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>300</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>开始计算</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pBot3">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>300</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>清空</string>
   </property>
  </widget>
  <widget class="QComboBox" name="box2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>90</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>按贷款总额计算</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>按面积和单价计算</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="l1">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>贷款类别：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line1">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>130</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="line2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>170</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pBot2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>300</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
  </widget>
  <widget class="QLabel" name="l_5">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>210</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
  <widget class="QLabel" name="l5">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>170</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>按揭年数：</string>
   </property>
  </widget>
  <widget class="QLabel" name="l8">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>250</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>还款方式：</string>
   </property>
  </widget>
  <widget class="QLabel" name="l7">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>210</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>贷款利率：</string>
   </property>
  </widget>
  <widget class="QLabel" name="l_1">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>130</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>(元）</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rBot1">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>250</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>等额本息</string>
   </property>
  </widget>
  <widget class="QLabel" name="l_2">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>170</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>（年）</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rBot2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>250</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>等额本金</string>
   </property>
  </widget>
  <widget class="QLabel" name="l3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>130</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>贷款总额：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line3">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>210</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pBot2</sender>
   <signal>released()</signal>
   <receiver>totalwidge</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>311</y>
    </hint>
    <hint type="destinationlabel">
     <x>210</x>
     <y>207</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
